app/views/passengers/_form.html.haml

```diff
@@ -9,8 +9,9 @@
 = form_with model: @passenger, local: true do |form|
   .field
     = form.label :permanent
-    = form.check_box :permanent, disabled: @current_user.dispatcher?,
-      id: :passenger_permanent
+    - unless @current_user.dispatcher?
+      = form.check_box :permanent, disabled: @current_user.dispatcher?,
```

Comment:
Why disable it if the current user is a dispatcher while in a block that won't run if the current user is a dispatcher?

Reply:
This is what @mboneil10 meant by

> Probably safe to remove disabling the checkbox logic?

Reply:
I see now! Yes.